shared/local-debug.desktop.js

```diff
@@ -7,7 +7,6 @@ import * as Tabs from './constants/tabs'
 import {updateConfig} from './command-line.desktop.js'
 
 let config = {
-  skipLoginRouteToRoot: false,
```

Comment:
instead of a flag which tries to keep what we init here untouched, let's just override the tab to go to when you log in